### PR TITLE
Make ykcbf compile again.

### PR DESCRIPTION
**Pull request by vext01**
Runs as far as stop-gapping.

### CODE REVIEW

File: bf_simple_yk.c [hunk 1]
```diff
@@ -103,12 +105,15 @@ int main(int argc, char *argv[]) {
     if (cells == NULL)
         err(1, "out of memory");
 
+    YkMT *mt = yk_mt_new();
+    yk_mt_hot_threshold_set(mt, 5);
```
Comment (ltratt) [inline bf_simple_yk.c]: Do we need to change the threshold?

Comment (vext01) [reply bf_simple_yk.c]: Probably not actually. Will revert.

Comment (vext01) [reply bf_simple_yk.c]: Fixed in [6522d09](https://github.com/ykjit/ykcbf/pull/10/commits/6522d09fd5d247f09605de37bcaab82324d845ad)

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**